pkgs/by-name/de/desk-exec/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  installShellFiles,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "desk-exec";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "AxerTheAxe";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-bJLdd07IAf+ba++vtS0iSmeQSGygwSVNry2bHTDAgjE=";
+  };
+
+  cargoHash = "sha256-pK1WDtGkpPZgFwEm59TqoH7EkKPQivNuvsiOG0dbum4=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion target/${stdenv.targetPlatform.config}/release/dist/desk-exec.{bash,fish}
+    installManPage target/${stdenv.targetPlatform.config}/release/dist/desk-exec.1
```

```suggestion
    installShellCompletion target/${stdenv.hostPlatform.config}/release/dist/desk-exec.{bash,fish}
    installManPage target/${stdenv.hostPlatform.config}/release/dist/desk-exec.1
```

See https://nixos.org/manual/nixpkgs/unstable/#ssec-cross-platform-parameters.

Or better:

```suggestion
    pushd target/${stdenv.hostPlatform.config}/release/dist
      installShellCompletion desk-exec.{bash,fish}
      installManPage desk-exec.1
    popd
```